### ISSUE TITLE
homepage is 404

### ISSUE THREAD

**Issue by ilovezfs** [imageworks/OpenColorIO#499]
```
Josephs-MacBook-Pro:~ joe$ curl -IL http://opencolorio.org
HTTP/1.1 404 Not Found
Server: GitHub.com
Date: Sat, 13 Jan 2018 09:22:31 GMT
Content-Type: text/html; charset=utf-8
Content-Length: 9116
Vary: Accept-Encoding
ETag: "590a28b8-239c"
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; img-src data:; connect-src 'self'
X-GitHub-Request-Id: A8E7:7582:35FFBE2:4B3106C:5A59CFD7

Josephs-MacBook-Pro:~ joe$
```

**Comment by retokromer** [comment]
FYI: Sony has been informed.

**Comment by scoopxyz** [comment]
Issue rectified and PR submitted for long-term fix #502

**Comment by ilovezfs** [comment]
Thanks @scoopxyz!
